diff --git a/node_modules/@satorijs/adapter-discord/.bun-tag-87bcbc389c8c3965 b/.bun-tag-87bcbc389c8c3965
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/lib/types/guild-member.d.ts b/lib/types/guild-member.d.ts
index 4b3138e7901e17afbb2445873f2e09573a1f6fd0..2a843ab2c1a7d0c042049c7e0096c7862b08e543 100644
--- a/lib/types/guild-member.d.ts
+++ b/lib/types/guild-member.d.ts
@@ -56,17 +56,19 @@ export declare namespace GuildMember {
         /** https://discord.com/developers/docs/resources/guild#modify-guild-member-json-params */
         interface Modify {
             /** value to set user's nickname to */
-            nick: string;
+            nick?: string | null
             /** array of role ids the member is assigned */
-            roles: snowflake[];
+            roles?: snowflake[] | null
             /** whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel */
-            mute: boolean;
+            mute?: boolean | null
             /** whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel */
-            deaf: boolean;
+            deaf?: boolean | null
             /** id of channel to move user to (if they are connected to voice) */
-            channel_id: snowflake;
+            channel_id?: snowflake | null
             /** when the user's timeout will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout */
-            communication_disabled_until?: timestamp;
+            communication_disabled_until?: timestamp | null
+            /** guild member flags */
+            flags?: integer | null
         }
         /** https://discord.com/developers/docs/resources/guild#modify-current-member-json-params */
         interface ModifyCurrent {
diff --git a/lib/types/message.d.ts b/lib/types/message.d.ts
index 2837a3175a671ccba95cc3f8cd1cb43c07e4a232..aac52729f75230635c2899e1c1c6e3d297644f91 100644
--- a/lib/types/message.d.ts
+++ b/lib/types/message.d.ts
@@ -159,11 +159,11 @@ export declare namespace Message {
     /** https://discord.com/developers/docs/resources/channel#create-message-jsonform-params */
     interface CreateParams extends EditParams {
         /** true if this is a TTS message */
-        tts: boolean;
+        tts?: boolean;
         /** include to make your message a reply */
-        message_reference: Reference;
+        message_reference?: Reference;
         /** IDs of up to 3 stickers in the server to send in the message */
-        sticker_ids: snowflake[];
+        sticker_ids?: snowflake[];
     }
     /** https://discord.com/developers/docs/resources/channel#edit-message-jsonform-params */
     interface EditParams {
